fbpic/openpmd_diag/data_dict.py

```diff
@@ -15,6 +15,8 @@
     "J" : np.array([-3., 1., 0., 1., 0., 0., 0.]),
     "E" : np.array([ 1., 1.,-3.,-1., 0., 0., 0.]),
     "B" : np.array([ 0., 1.,-2.,-1., 0., 0., 0.]),
+    "A" : np.array([ 1., 1.,-2.,-1., 0., 0., 0.]),
+    "dtA" : np.array([ 1., 1.,-3.,-1., 0., 0., 0.]),
```

Comment:
In fact, A (as stored in the code, and written in the file) is dimensionless, unlike the actual vector potential. (Unless I misunderstood your implementation)

Therefore, I would say that this should be:
```
    "A" : np.array([ 0., 0., 0., 0., 0., 0., 0.]),
    "dtA" : np.array([ 0., 0., -1., 0., 0., 0., 0.]),
```
Do you agree?